ShareTheTable/View Controllers/Profile Making/Add Photos/AddPhotosViewController.m

```diff
@@ -0,0 +1,137 @@
+//
+//  AddPhotosViewController.m
+//  ShareTheTable
+//
+//  Created by Andrea Gonzalez on 7/7/22.
+//
+
+#import "AddPhotosViewController.h"
+#import <Photos/Photos.h>
+#import <PhotosUI/PhotosUI.h>
+#import "SceneDelegate.h"
+#import "CreateAccountViewController.h"
+#import "QuestionViewController.h"
+#import "UniformTypeIdentifiers/UniformTypeIdentifiers.h"
+#import "Parse/Parse.h"
+
+@interface AddPhotosViewController () <PHPickerViewControllerDelegate>
+
+
+@end
+
+@implementation AddPhotosViewController
+
+- (IBAction)tapNext:(id)sender {
```

Comment:
tapNext & tapCancel do the same thing - you can refactor them into one separate function that takes in an identifier. Anytime you see yourself copy/pasting or doing the same thing twice, try to think of a way to simplify.